### PR TITLE
Initialize Board

### PR DESCRIPTION
**Pull request by VictorCespedesCC**
Se implemento la logica para la creacion de un tablero, la identificacion de celdas vacias y la  creacion de numeros randomicos.

### CODE REVIEW

**Comment by victor-villca** [comment]
@VictorCespedesCC Please review if your commits are ok, because no table creation logic, empty cells indentificatoin or random number creation was found.

**Review by victor-villca — CHANGES_REQUESTED** [review]
Nothing was Implemented

**Comment by VictorCespedesCC** [comment]
@victor-villca perdon el visual studio no guarda los archivos automaticamente, ya esta subido los cambios

**Review by victor-villca — CHANGES_REQUESTED** [review]
The implementation has erros, mostly the way you are importing and exporting methods. For example, i the logic file, you forgot to import the Data.Array module
image

**Comment by VictorCespedesCC** [comment]
se trataba de un import que faltaba en el archivo logic. done

**Review by victor-villca — CHANGES_REQUESTED** [review]
The LogicSpecTests have compilation errors
image

**Comment by VictorCespedesCC** [comment]
faltaba import en la parte de los tests

**Review by victor-villca — APPROVED** [review]
The test cases work as expeceted.

**Review by DeidamiaFuentes — COMMENTED** [review]
When I do stack run it doesn't show the board

image

File: src/Rendering.hs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+module Rendering where
+
+import Graphics.Gloss
+
+import Game
+import Data.Array
+
+boardGridColor = makeColorI 255 255 255 255
```
Comment (MayerliSantander) [inline src/Rendering.hs]: Change the color of the grid because it is not visible since the background is the same color.

**Comment by VictorCespedesCC** [comment]
the background color and the grid lines color are the same, sorry

**Review by DeidamiaFuentes — APPROVED** [review]
I thing everything is fine

**Review by MayerliSantander — APPROVED** [review]

**Pull request merged**